chapter02/spg/question3.pl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Business::ISBN;
+
+my $isbn = Business::ISBN->new('9781449393090');
+
+print "Printing data for ISBN 9781449393090", "\n";

```

Comment:
It'd probably more resilient to change if the ISBN were maintained as a single variable and interpolated into this print statement. Alternatively, you can use the `Business::ISBN::as_string` method here to show the ISBN (formatted with hyphens).